apps/src/templates/manageStudents/MoveStudents.story.jsx

```diff
@@ -40,35 +44,60 @@ const transferToOtherTeacher = {
   copyStudents: false
 };
 
+const errorTransferStatus = {
+  status: TransferStatus.FAIL,
+  error: 'You cannot move these students because they are already in the new section.'
+};
+
 export default storybook => {
   storybook
     .storiesOf('MoveStudents', module)
     .addStoryTable([
       {
         name: 'Move students dialog',
-        description: 'Ability to move students in a certain section to a different section or teacher',
+        description: 'Empty dialog',
         story: () => (
           <MoveStudents
             studentData={studentData}
             transferData={blankStudentTransfer}
+            transferStatus={blankStudentTransferStatus}
+            currentSectionId={1}
+            sections={sections}
+            updateStudentTransfer={() => console.log('updating...')}
+            transferStudents={() => console.log('transferring...')}
+            cancelStudentTransfer={() => console.log('cancelling...')}
```

Comment:
There's a storybook "actions" addon designed for stubbing callbacks like this, that logs to the onscreen console (not the developer console) and reports any arguments passed to the callback as well.  Example:

https://github.com/code-dot-org/code-dot-org/blob/734199bb62fd835f58b88176f0f008692a175380/apps/src/lib/ui/ConfirmEnableMakerDialog.story.jsx#L3-L17